dev-docs/bidders/between.md

```diff
@@ -15,6 +15,9 @@ tcf2_supported: true
 {: .table .table-bordered .table-striped }
 | Name          | Scope    | Description | Example | Type     |
 |---------------|----------|-------------|---------|----------|
-| `host` | required |  between host url prefix         | `eu`   | `string` |
+| `host` | required |  Between SSP host url prefix. Defines data center where requests will be sent. Choose the closest one to the prebid-server you are using. Allowed values: `lbs-eu1.ads`, `lbs-ru1.ads`, `lbs-asia1.ads`, `lbs-us-east1.ads` | `lbs-eu1.ads` | `string` |
```

Comment:
For the record I'm not sure this parameter is very effective. The params here are set by the publisher for all users. Which means a given pub chooses one-and-only-one endpoint for all of their users. 

A better way to manage endpoint mapping is in the host company's prebid server config. e.g. AppNexus or Magnite can tie their european clusters to your lbs-eu1.ads endpoint, their US clusters to your lbs-us-east1.ads endpoint, etc.

Reply:
>A better way to manage endpoint mapping is in the host company's prebid server config. e.g. AppNexus or Magnite can tie their european clusters to your lbs-eu1.ads endpoint, their US clusters to your lbs-us-east1.ads endpoint, etc.

As far as I know, prebid server does not have such a mechanism. The `host` parameter is used by many others adapters. 
See: [AcuityAds Adapter](https://docs.prebid.org/dev-docs/pbs-bidders.html#acuityads), [AdKernel Adapter](https://docs.prebid.org/dev-docs/pbs-bidders.html#adkernel),  [AndBeyond Adapter](https://docs.prebid.org/dev-docs/pbs-bidders.html#andbeyond), etc.
So except if prebid server provider use DNS-balancing, the host parameter can not affect users in any way - it determines endpoint for server-to-server requests only